app/uk/gov/hmrc/cgtpropertydisposalsfrontend/views/account/manage_your_details.scala.html

```diff
@@ -37,10 +29,11 @@
  cyaChange: uk.gov.hmrc.cgtpropertydisposalsfrontend.views.html.components.cya_change,
  addressDisplay: uk.gov.hmrc.cgtpropertydisposalsfrontend.views.html.components.address_display,
  submitButton: uk.gov.hmrc.cgtpropertydisposalsfrontend.views.html.components.submit_button,
- warningMessage: uk.gov.hmrc.cgtpropertydisposalsfrontend.views.html.components.warning
+ warningMessage: uk.gov.hmrc.cgtpropertydisposalsfrontend.views.html.components.warning,
+ alertBanner: uk.gov.hmrc.cgtpropertydisposalsfrontend.views.html.components.alert_banner
 )
 
-@(details: SubscribedDetails)(implicit request:Request[_], messages:Messages, appConfig: ViewConfig)
+@(details: SubscribedDetails, detailChanged: Option[SubscriptionDetail])(implicit request:Request[_], messages:Messages, appConfig: ViewConfig)
```

Comment:
`detailChanged: Option[SubscriptionDetail])` 

should be called something like

`updatedDetails`: Option[UpdatedSubscribedDetails]`

Reply:
I named `SubscriptionDetail` that to be as generic as possible - we could reuse for something other than indicating that a certain field has changed. It's also singluar here since you can have only changed one thing.

Reply:
If you did that, then your implementation will be very confusing to read eventually. A person wouldn't know that a reference to SubscriptionDetail was to do with a change in detail or something entirely different. It's not a good approach to take.

Reply:
Actually, the name is fine. I took another look at the defn of SubscriptionDetail

Reply:
:+1:  the context should be able to tell you what it's being used for